runtime/jcl/common/proxy.c

```diff
@@ -65,6 +78,11 @@ static jclass proxyDefineClass(
 
 	vmFuncs->internalExitVMToJNI(currentThread);
 
+#if JAVA_SPEC_VERSION == 8
+	if (isProxyExemptFromVerification(env, className)) {
+		return defineClassCommon(env, classLoader, className, classBytes, offset, length, pd, J9_FINDCLASS_FLAG_UNSAFE, NULL);
+	}
+#endif /* JAVA_SPEC_VERSION == 8 */
 	return defineClassCommon(env, classLoader, className, classBytes, offset, length, pd, 0, NULL);
```

Comment:
These two calls only differ in one parameter.  To make that clearer, can you refactor the code like:
```
UDATA defineClassOptions = 0;
....
if (isProxyExemptFromVerification(env, className)) {
    defineClassOptions |= J9_FINDCLASS_FLAG_UNSAFE;
}
return defineClassCommon(env, classLoader, className, classBytes, offset, length, pd, defineClassOptions, NULL);
```